src/utils/dom.spec.ts

```diff
@@ -119,11 +123,17 @@ describe("dom", () => {
         </h2>
         <h2 slot=${testSlotName}><span>😂</span></h2>
         <h3 slot=${testSlotName2}><span>😂</span></h3>
+        <div id="default-slot-el"><p>🙂</p></div>
       </slot-test>
     `;
+
+      const assignedSlot = document.querySelector("slot-test").shadowRoot.querySelector(`slot:not([name])`);
```

Comment:
Is this a workaround for `Node.assignedSlot` missing? If so, just wanted to check if it's working or not based on your comments to move to an e2e test.

Reply:
Yes, this is a workaround.

Reply:
Without this, the assignedNode won't be there. So we would have to run the getSlotted() method in the browser. Which means placing the whole utility in the browser so that all methods can be accessed.